frontend/src/app/application-dashboard/components/csi-value/csi-value.component.spec.ts

```diff
@@ -165,7 +165,7 @@ describe('CsiValueComponent', () => {
     component.csiValue = goodValue;
     component.isBig = false;
     component.csiDate = new Date("01.02.2017").toISOString();
-    component.lastResultDate = new Date("01.03.2017").toISOString();
+    component.lastResultDate = new Date("02.02.2017").toISOString();
```

Comment:
why did change a variable within a test spec which is not related to your issue?

Reply:
It was just a test and I forgot to undo this change.